sed/config/flash_example_config.yaml

```diff
@@ -3,20 +3,26 @@
 core:
   # defines the loader
   loader: flash
-
   # the beamline where experiment took place
   beamline: pg2
-
-  # The paths to the raw and parquet data directories.
+  # the instrument used
+  instrument: hextof
+  # the ID number of the beamtimme
+  beamtime_id: 11013410
+  # the year of the beamtime
+  year: 2023
+
+  # The paths to the raw and parquet data directories. If these are not
+  # provided, the loader will try to find the data based on year beamtimeID etc
   paths:
+    # location of the raw data.
     data_raw_dir: "tests/data/loader/flash/"
+    # location of the intermediate parquet files.
     data_parquet_dir: "tests/data/loader/flash/parquet"
 
-  # These can be replaced by beamtime_id and year to automatically
-  # find the folders on the desy cluster
-
-  # beamtime_id: xxxxxxxx
-  # year: 20xx
+binning:
+  # Since this will run on maxwell most probably, we have a lot of cores at our disposal
```

Comment:
I believe including the config file link from documentation in the notebook would be good. As this file isnt really accesible to users so trivially (when they install with pip)

Reply:
The flash example config should be also included if you install via pip